content/posts/blog/2024-03-13-image-and-distribution-1-1.md

```diff
@@ -0,0 +1,209 @@
+---
+title: OCI Image and Distribution Specs v1.1 Releases
+author:
+  name: Open Container Initiative
+  tag: oci
+tags: blog
+date: 2024-03-13
+---
+
+The OCI Image Specification and Distribution Specification each had a 1.1.0 release on February 15, 2024.
+These are the first minor releases since the 1.0.0 releases in July 2017 and May 2021 respectively, and include a number of notable changes.
+
+The biggest highlight of these releases is the process for associating metadata with existing images using a new `subject` field in the image specification and the associated referrers API in the distribution specification.
+However, this is far from the only thing that was changed.
+And many of these changes may affect existing OCI conformant projects.
+
+# Guidance for Artifacts
+
+OCI has been used to package objects other than container images for some time, including projects like Helm, OPA, and WASM.
+The guidance for how projects should implement artifacts has now been [codified into the image specification][image-artifact-guidance].
+
+The recommendation to use a `config.mediaType` value different from the recognized container image values remains.
+That guidance is now extended to include a new `artifactType` field that enables use cases without a dedicated `config.mediaType`.
+In the scenarios were an artifact does not have content for the config or any layers, an empty JSON descriptor is recommended as a placeholder to satisfy the requirement that these fields exist, while serving as an indication to consuming tools that there is no useful content in that descriptor.
```

Comment:
a link to the "empty JSON descriptor"? ... an important enough constant to call out?

Reply:
https://github.com/opencontainers/image-spec/blob/main/manifest.md#guidance-for-an-empty-descriptor

Reply:
Added, good call out!